src/ar_util.erl

```diff
@@ -40,13 +40,26 @@ pick_random(Xs) ->
 encode(Bin) ->
 	base64url:encode(Bin).
 
-%% @doc Decode a URL safe base64 to binary.
+%% @doc Try to decode a URL safe base64 into a binary or throw an error when
+%% invalid.
 decode([]) -> [];
```

Comment:
This is old code, I know, but my god how unexpected to get a string back (instead of a binary) if the argument is an empty string.

Reply:
Are you suggesting we change that behaviour?